packages/sit-onyx/src/components/OnyxDataGrid/features/selection/selection.ts

```diff
@@ -18,6 +18,7 @@ export const useSelection = createFeature(
           contingent: new Set<TEntry["id"]>(),
         } as const),
     );
+    const rowsCount = ref();
```

Comment:
If we initialize it with a number value, the type will be `Ref<number>` instead of `Ref<number | undefined>`
```suggestion
    const rowsCount = ref(0);
```